generated/nifake/tests/test_session.py

```diff
@@ -284,6 +284,47 @@ def test_cannot_add_properties_to_session(self):
             except AttributeError as e:
                 pass
 
+    def test_get_vi_real64_attribute(self):
```

Comment:
This is independent of this PR right?

Reply:
It is. Which is why I requested changes on other PRs so things like this aren't required any more